sources/packages/web/src/views/aest/student/StudentAccountApplications.vue

```diff
@@ -4,10 +4,20 @@
       <header-navigator title="Student requests" subTitle="Accounts" />
     </template>
     <body-header
-      title="Requested accounts"
-      subTitle="Make a determination for students requesting to login with a Basic BCeID."
+      title="Pending account requests"
+      subTitle="Basic BCeID account requests that require ministry review."
       :recordsCount="accountApplications?.length"
     >
+      <template #actions>
+        <v-text-field
+          density="compact"
+          label="Search first name or last name"
```

Comment:
I'm not sure if we're going to keep this but should it be "Search given names or last name"?